tools/build/scripts/utils/recursiveReaddir.js

```diff
@@ -0,0 +1,96 @@
+const fs = require("fs");
```

Comment:
mind adding a link back to the original file at the top of this file to make it easily trace-able. We do this in a few other places as well.